lib/jekyll-sitemap.rb

```diff
@@ -23,15 +29,9 @@ def generate(site)
       end
     end
 
-    HTML_EXTENSIONS = %W(
-      .html
-      .xhtml
-      .htm
-    )
-
-    # Array of all non-jekyll site files with an HTML extension
-    def html_files
-      @site.static_files.select { |file| HTML_EXTENSIONS.include? file.extname }
+    # Array of all non-jekyll site files with given extensions
+    def sitemap_static_files (exts=[])
```

Comment:
Since this is in the `JekyllSitemap` class, the name feels redundant. Could we just call this `static_files` instead?